memory/transparent_hugepages.py

```diff
@@ -48,6 +49,11 @@ def setUp(self):
         self.mem_path = os.path.join(data_dir.get_tmp_dir(), 'thp_space')
         free_mem = int(memory.freememtotal() / 1024)
         self.dd_timeout = 900
+        self.thp_split = None
+        if distro.detect().name == "rhel":
+            self.thp_split = "thp_split"
+        else:
+            self.thp_split = "thp_split_page"
```

Comment:
The above can also be implemented in a simple way.
```
self.thp_split = "thp_split_page"
if distro.detect().name == "rhel":
    self.thp_split = "thp_split"
```

Reply:
Sure